requirements.txt

```diff
@@ -2,31 +2,31 @@
 boto3>=1.4.2
 click>=6.6
 jsonschema>=2.4
+python-dateutil>=2.5.3
```

Comment:
I'm fairly reluctant to pin with a `>=`; we should probably pin to specific minor-release versions of packages, e.g. using ~=, to prevent unexpected downstream breakages from taking out this library.